docker-compose.yml

```diff
@@ -6,9 +6,11 @@ services:
       - ./repositories:/repositories
       - ./query:/query
     environment:
-      DB_HOST: scylla
+      HOST: scylla
       BBLFSH_HOST: bblfshd
       FEATURES_EXTRACTOR_HOST: featurext
+    ports:
```

Comment:
one of the reasons why there are no ports - make sure user can run gemini without port collisions.

Reply:
I see. But without having this we would not be able to point users of `docker-compose` to Spark web ui, which is crucial for debugging.

Reply:
I would suggest such users use `.host.yaml` config. But okay. The only problem I'll have to edit `docker-compose.yml` every time when I want to test it with docker-compose but I can live with it.